src/csharp/Microsoft.Spark/JvmException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Spark
+{
+    public class JvmException : Exception
+    {
+        private readonly string _jvmFullException;
+
+        public JvmException(string jvmFullException)
+        {
+            _jvmFullException = jvmFullException;
+        }
+
+        public override string Message => _jvmFullException;
```

Comment:
You can just have
```C#
public JvmException(string message)
    : base(message)
{
}
```
and not implement this.

Reply:
Done